migrations/20170913003259_user_is_email_notifications.js

```diff
@@ -0,0 +1,11 @@
+export async function up(knex) {
+  await knex.schema.table('users', table => {
+    table.boolean('is_email_notifications').notNullable().defaultTo(false);
```

Comment:
We should think about better name for this field. Not to mention that it should not be limited to `true/false`, but should be compatible with wider set of values.

Either it should be an ENUM, or we could even use some kind of jsonb-field here (something like #237)

Reply:
Of course it's not very good name for this field, but @dsumin wants this field to be boolean and named in that way.

Reply:
@indeyets as there are no set plans for extending this feature I would suggest to leave it as is. Changing to jsonb when necessary.